WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -483,28 +486,44 @@ import WordPressFlux
 
     // MARK: - Setup
 
-    private func setupTableView() {
+    private func setupStackView() {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+
+        setupTableView(stackView: stackView)
+        setupJetpackBanner(stackView: stackView)
+
+        view.addSubview(stackView)
+        view.pinSubviewToAllEdges(stackView)
+    }
+
+    private func setupJetpackBanner(stackView: UIStackView) {
+        guard AppConfiguration.isWordPress && FeatureFlag.jetpackPowered.enabled else {
+            return
+        }
+
+        let jetpackBannerView = JetpackBannerView()
+        jetpackBannerView.setup()
+        addTranslationObserver(jetpackBannerView)
+        stackView.addArrangedSubview(jetpackBannerView)
+        jetpackBannerView.heightAnchor.constraint(greaterThanOrEqualToConstant: 50).isActive = true
```

Comment:
Normally I would be bitching about putting the `50` constant in a group of constants 🤣, but looking at this file, it would probably just complicate it further.

Reply:
@Gio2018 what about making it a static var for the JetpackBannerView? e749c114ea7099ee4fded14a81d624cab06a9810

Reply:
That sounds good to me!